src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -2399,7 +2399,8 @@ namespace winrt::Microsoft::Terminal::Control::implementation
 
     void TermControl::_TerminalTitleChanged(const std::wstring_view& wstr)
     {
-        _titleChangedHandlers(winrt::hstring{ wstr });
+        auto titleArgs = winrt::make_self<TitleChangedEventArgs>(winrt::hstring{ wstr });
+        _TitleChangedHandlers(*this, *titleArgs);

```

Comment:
something future peeps can do:

```suggestion
        _TitleChangedHandlers(*this, winrt::make<TitleChangedEventArgs>(winrt::hstring{ wstr }));
```

that's the purpose of make (without self)! :smile:

Reply:
_i am pretty sure_

Reply:
I can check

Reply:
yep you definitely can